packages/date-picker/src/_subcomponents/Calendar/Calendar.module.scss

```diff
@@ -45,6 +45,11 @@ $selectors--button-focus: "&:focus-visible, #{$polyfill--focus-visible}";
   }
 }
 
+// UX has been updated such that disabled days do not receive focus,
+// and keyboard navigation only lands on available days.
+// If this changes at some point again, these styles are available for use.
+// https://github.com/gpbl/react-day-picker/pull/1519
+// https://github.com/gpbl/react-day-picker/issues/1449#issuecomment-1149942033
 %focus-ring--disabled {
   @extend %focus-ring;
```

Comment:
I think we should delete the styles but leave the comment? It's definitely a micro-optimisation, but I want to avoid bundling unused CSS.

Reply:
Yeah I was considering that too. Happy to remove it :)